metadata/src/main/java/org/apache/kafka/metadata/storage/Formatter.java

```diff
@@ -348,20 +352,20 @@ private short effectiveKRaftFeatureLevel(Optional<Short> configuredKRaftVersionL
             if (configuredKRaftVersionLevel.get() == 0) {
                 if (hasDynamicQuorum()) {
                     throw new FormatterException(
-                        "Cannot set kraft.version to " +
-                        configuredKRaftVersionLevel.get() +
-                        " if one of the flags --standalone, --initial-controllers, or --no-initial-controllers is used. " +
-                        "For dynamic controllers support, try removing the --feature flag for kraft.version."
+                        "Cannot set kraft.version to " + configuredKRaftVersionLevel.get() +
```

Comment:
Would you mind using `0` instead of `configuredKRaftVersionLevel.get()`? for example:
```java
                    throw new FormatterException(
                        "Cannot set kraft.version to 0 if controller.quorum.voters is empty and one of the flags --standalone, " +
                        "--initial-controllers, or --no-initial-controllers is used. For dynamic controllers support, " +
                        "try removing the --feature flag for kraft.version."
                    );
```